src/resources.js

```diff
@@ -813,9 +913,15 @@ export class Resources {
     }
     if (request) {
       request.newHeight = newHeight;
+      request.force = force || request.force;
+      request.fallback = fallback || request.fallback;
     } else {
-      this.changeHeightRequests_.push(
-          {resource: resource, newHeight: newHeight});
+      this.changeHeightRequests_.push({
+        resource: resource,
+        newHeight: newHeight,
+        force: force,
+        fallback: fallback
+      });
```

Comment:
yep, this should work with babel-core (no helpers required). not sure if closure-compiler has support for it, which is why i hazard supporting us to use it :smile_cat: